apps/microsoft-teams/frontend/src/components/config/NotificationsSection/NotificationsSection.tsx

```diff
@@ -1,17 +1,28 @@
+import { Dispatch } from 'react';
 import { Box, Subheading } from '@contentful/f36-components';
 import { styles } from './NotificationsSection.styles';
 import { notificationsSection } from '@constants/configCopy';
 import AddButton from '@components/config/AddButton/AddButton';
 import NotificationEditMode from '@components/config/NotificationEditMode/NotificationEditMode';
 import { Notification } from '@customTypes/configPage';
+import { ParameterAction, actions } from '@components/config/parameterReducer';
 
 interface Props {
   notifications: Notification[];
-  createNewNotification: () => void;
+  dispatch: Dispatch<ParameterAction>;
 }
 
 const NotificationsSection = (props: Props) => {
-  const { notifications, createNewNotification } = props;
+  const { notifications, dispatch } = props;
+
+  const createNewNotification = () => {
+    dispatch({ type: actions.ADD_NOTIFICATION });
+  };
+
+  const deleteNotification = (index: number) => {
+    notifications.splice(index, 1);
```

Comment:
I feel we may want to either filter on the array or create a new one here. 
Eg.
```js 
const notificationsPayload = [...notifications];
notifcationsPayload.splice(index, 1);
```

Main reason being it's best practice in react to not mutate state directly. While this should hopefully be fine since we're dispatching right after, it could cause some unintended side-effects

Reply:
Great catch, updated!